x-pack/plugins/cases/public/components/markdown_editor/utils.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const getMarkdownEditorStorageKey = (
+  appId: string,
+  caseId: string,
+  commentId: string
+): string => {
+  const appIdKey = appId !== '' ? appId : 'kibana';
```

Comment:
Let's default it to `cases` instead of `kibana`.

Reply:
We are returning key as `return cases.${appIdKey}.${caseIdKey}.${commentIdKey}.markdownEditor` so if we put `cases` when no `appId`, it will be `cases.cases.${caseId}.${commentId}.markdownEditor`

Reply:
shall I put `const appIdKey = appId !== '' ? appId : 'default';` ?

Reply:
I think is better to put `const appIdKey = appId !== '' ? appId : 'cases';`